src/lib/router/definitions.ts

```diff
@@ -20,7 +20,14 @@ export interface ProjectsParams {
         params?: {
           view: { resource: "new" };
         };
-      };
+      }
+    | {
+        resource: "patches";
+        params?: {
+          view: { resource: "new" };
+        };
+      }
+    | { resource: "patch"; params: { patch: string } };
```

Comment:
Feel like it should rather be:
```suggestion
    | { resource: "patch"; params: { id: string } };
```

And the same for issues.

Reply:
hmm we use `id` currently for `project` ids, not sure if we gain much by changing this, we would need to change `id` everywhere to `id: project` or something.

Reply:
Ah, right.